docs/source/upgrade-guide/index.md

```diff
@@ -240,6 +240,14 @@ This vision was partially implemented in Volto, bypassing the information coming
 Since this never happened, we are going back to the classic mode, so the dropdown will show the transition names.
 When the simplified vision is implemented, we will revisit it.
 
+#### Move Layout constants to `config.views.layoutViewsNamesMapping`.
+
+The `constants` layout module was removed in favor of an object in the Configuration Registry: `config.views.layoutViewsNamesMapping`.
+
+If you have added or modified the Plone layout views literal mapping, you should use now this setting, and you can remove the module shadowing customization.
```

Comment:
@sneridagh It would be helpful to include an example of the setting to show how it is structured

Reply:
@sneridagh Also in general, I would expect that a new setting also needs to be documented somewhere other than in the upgrade docs?

Reply:
Done